src/main/java/io/github/sskorol/model/OperationStatus.java

```diff
@@ -0,0 +1,20 @@
+package io.github.sskorol.model;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Enum for purchase and login statuses.
+ */
+
+@Getter
+@RequiredArgsConstructor
+public enum OperationStatus {
+
+    PURCHASE_SUCCESSFUL("Operation was successfully completed"),
+    PURCHASE_FAILED("It is impossible to complete purchase operation"),
+    LOGIN_SUCCESSFUL("Authorized successfully"),
+    LOGIN_FAILED("Cannot find an account with such credentials");
+
+    private final String operationStatus;
```

Comment:
It makes no sense to call your instance var the same as actual class, because when you call getter, it looks a bit strange:
```java
LOGIN_FAILED.getOperationStatus();
```
Would be much more obvious to use `name` or smth similar.